test.py

```diff
@@ -505,7 +510,8 @@ def start_test_background(framework, cluster, repo_root):
                 framework.name)
     _setup_strict(framework, cluster, repo_root)
 
-    logger.info("Launching shakedown for %s", framework.name)
+    logger.info("Launching shakedown in background for %s", framework.name)
+    logger.debug("stub_universe:{framework.stub_universe_url} cluster_url:{cluster.url} authtoken:{cluster.auth_token}".format(locals()))
```

Comment:
If you are keeping this debug and the one on line 546, please pick one style of formatting that string